lib/apt/build.sh

```diff
@@ -74,6 +74,8 @@ fi
 run mkdir -p ~/.lintian/profiles/td-agent/
 run cp "debian/lintian/td-agent/${distribution}.profile" ~/.lintian/profiles/td-agent/${distribution}.profile
 # export DEB_BUILD_OPTIONS=noopt
+DISTRIBUTION=`lsb_release -c | cut -f2`
+sed -i'' -e "s/unstable/$DISTRIBUTION/" debian/changelog
```

Comment:
This regular expression is too rough, all `unstable` string will be replaced.

Reply:
How about `sed -i'' -e "1s/ unstable;/ $DISTRIBUTION;/" debian/changelog` ?

Reply:
It's still rough, should be matched more strictly. And older version should be also replaced instead of replacing only latest version.
Like this?:

`sed -E "s/^($PACKAGE \(\S+\)) unstable;/\1 $DISTRIBUTION;/g" changelog`

Reply:
Thanks, It is better to restrict expression like above.

Reply:
Fixed.